src/test/java/com/iexec/worker/docker/DockerServiceTests.java

```diff
@@ -288,6 +282,52 @@ void shouldRunAndNotRemoveContainerFromRecordsSinceInDetachedMode() {
         verify(dockerService, never()).removeFromRunningContainersRecord(containerName);
     }
 
+    @Test
+    void shouldNotDisplayLogsWhenChainTaskIdNotFound(CapturedOutput output) {
+        DockerRunRequest dockerRunRequest = DockerRunRequest.builder()
+                .containerName("containerName").build();
+        DockerRunResponse successResponse = DockerRunResponse.builder().build();
+        doReturn(dockerClientInstanceMock).when(dockerService).getClient();
+        when(dockerClientInstanceMock.run(dockerRunRequest)).thenReturn(successResponse);
+        when(workerConfigService.isDeveloperLoggerEnabled()).thenReturn(true);
+
+        DockerRunResponse dockerRunResponse = dockerService.run(dockerRunRequest);
+        assertThat(dockerRunResponse).isNotNull();
+        assertThat(output.getOut()).contains("Cannot print developer logs");
+    }
+
+    @Test
+    void shouldDisplayLogs(CapturedOutput output) {
+        final String stdoutMessage = "Message written on stdout";
+        final String stderrMessage = "Message written on stderr";
+        DockerRunRequest dockerRunRequest = DockerRunRequest.builder()
+                .chainTaskId(CHAIN_TASK_ID)
+                .containerName("containerName").build();
+        DockerLogs dockerLogs = DockerLogs.builder()
+                .stdout("Message written on stdout")
+                .stderr("Message written on stderr")
```

Comment:
```suggestion
                .stdout(stdoutMessage)
                .stderr(stderrMessage)
```

Reply:
Oupsy, thanks for your awareness
Fixed in dbfceab